test/Microsoft.NET.Build.Tests/DeleteNuGetArtifactsFixture.cs

```diff
@@ -12,9 +12,15 @@ public class ConstantStringValues
         public static string NuGetSharedDirectoryNamePostfix = "_NuGetDependencies";
         public static string NetstandardToken = "netstandard";
         public static string DependencyDirectoryNamePrefix = "D_";
-        public static string NuGetPackageBaseDirectory = Path.Combine(
-            RepoInfo.GetBaseDirectory(),
-            TestDirectoriesNamePrefix + NuGetSharedDirectoryNamePostfix);
+        public static string NuGetPackageBaseDirectory
+        {
+            get
+            {
+                return Path.Combine(TestContext.Current.TestExecutionDirectory,
+                                    TestDirectoriesNamePrefix + NuGetSharedDirectoryNamePostfix);
```

Comment:
It's unfortunately pervasive in CLI/SDK code to do stuff like this in property getters, but we shouldn't produce fresh strings on every property access. If I review calling code that calls this more than once, I'm not going to imagine that to be problematic, because it looks like field access. Now, this is test code, so my care factor is not high, but it's something that I'm going to be pushing back on in product code.